File: src/main/java/Dado.java
public class Dado {
    private String resposta1 = "-----" +
                               "|   |" +
                               "| * |" +
                               "|   |" +
                               "-----";
    private String resposta2 = "-----" +
                               "|   |" +
                               "|* *|" +
                               "|   |" +
                               "-----";
    private String resposta3 = "-----" +
                               "| * |" +
                               "| * |" +
                               "| * |" +
                               "-----";
    private String resposta4 = "-----" +
                               "| * |" +
                               "|* *|" +
                               "| * |" +
                               "-----";
    private String resposta5 = "-----" +
                               "| * |" +
                               "|***|" +
                               "| * |" +
                               "-----";
    private String resposta6 = "-----" +
                               "|* *|" +
                               "|* *|" +
                               "|* *|" +
                               "-----";

    public String rolarDado() {
        int valor = (int)(Math.random() * 5) + 1;

        switch (valor) {
            case 1:
                return resposta1;
            case 2:
                return resposta2;
            case 3:
                return resposta3;
            case 4:
                return resposta4;
            case 5:
                return resposta5;
            case 6:
                return resposta6;
            default:
                return "erro";
        }
    }
}

File: src/main/java/Dado.kt
class Dado {
    private val resposta1 = "-----\n" +
            "|   |\n" +
            "| *|\n" +
            "|   |\n" +
            "-----"
    private val resposta2 = "-----\n" +
            "|   |\n" +
            "|**|\n" +
            "|   |\n" +
            "-----"
    private val resposta3 = "-----\n" +
            "| *|\n" +
            "| *|\n" +
            "| *|\n" +
            "-----"
    private val resposta4 = "-----\n" +
            "| * |\n" +
            "|**|\n" +
            "| * |\n" +
            "-----"
    private val resposta5 = "-----\n" +
            "| * |\n" +
            "|***|\n" +
            "| * |\n" +
            "-----"
    private val resposta6 = "-----\n" +
            "|**|\n" +
            "|**|\n" +
            "|**|\n" +
            "-----"

    fun rolarDado(): String {
        val valor = (Math.random() * 5).toInt() + 1

        when (valor) {
            1 -> return resposta1
            2 -> return resposta2
            3 -> return resposta3
            4 -> return resposta4
            5 -> return resposta5
            6 -> return resposta6
            else -> return "erro"
        }
    }
}
